src/components/builder.ts

```diff
@@ -48,24 +48,68 @@ export class Builder {
     }
 
     /**
-     * Kill the current building process.
+     * Stop the current build process.
      */
-    kill() {
-        const proc = this.currentProcess
-        if (proc) {
-            const pid = proc.pid
+    async stop() {
+        if (this.currentProcess) {
+            const envVarsPosix: ProcessEnv = {PATH: '/bin:/usr/bin'}
+            const envVarsWindows: ProcessEnv = {PATH: '%SystemRoot%\\system32;%SystemRoot%;%SystemRoot%\\System32\\Wbem'}
```

Comment:
I am against changing the `PATH` on Windows. I do not know how canonical they are on Windows and there is hardly no chance that there is an other `pkill` executable available in the `PATH`. Please, revert these changes.